bootstrap-salt.sh

```diff
@@ -23,7 +23,7 @@
 #======================================================================================================================
 set -o nounset                              # Treat unset variables as an error
 
-__ScriptVersion="2019.11.04"
+__ScriptVersion="2019.12.19"
```

Comment:
Is it supposed to be changes on releases, isn't it?

Reply:
Agreed, please don't _bump_ the version.

Reply:
Whoops, reverted back to original value, got carried away with update, similar to AIX / Solaris build scripts which are updated whenever touched.